.github/workflows/test-update-contribs.yml

```diff
@@ -42,6 +42,8 @@ jobs:
           commit-message: "Update: Contributor & review file update"
           delete-branch: true
           title: Update contributor and review data
+          labels: |
```

Comment:
this is perfect. I presume this will trigger the bot to fix the pr? i had no idea this was an option.